paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestAuthorization.kt

```diff
@@ -54,7 +54,7 @@ class TestAuthorization {
     }
 
     private val bancontactNewUser = TestParameters(
-        SupportedPaymentMethod.Bancontact,
+        lpmRepository.fromCode("bancontact")!!,
```

Comment:
Why can't we keep this as `SupportedPaymentMethod.Bancontact`

Reply:
SupportedPaymentMethod no longer has hard coded Lpms, they are read from disk and stored in a map.